setup.cfg

```diff
@@ -32,7 +32,7 @@ install_requires =
     psycopg2
     django-better-admin-arrayfield
     django-solo
-    mozilla-django-oidc
+    mozilla-django-oidc >=1.0.0, <2.0.0
```

Comment:
@sergei-maertens when installing `mozilla-django-oidc-db`, I noticed that `mozilla-django-oidc==2.0.0` was being installed, which is incompatible. Would pinning like this be the proper way to deal with that for now (until I actually fix the compatibility issues)?

Reply:
yeah, that should work. You can also use `~=1.0` I believe, which should give you anything like `1.0, 1.1, 1.2` etc.